web/modules/custom/dbcdk_community/templates/dbcdk-community--flagged-content--content.html.twig

```diff
@@ -0,0 +1,36 @@
+{#
+/**
+ * @file
+ * Theme to present DBCDK Community FlaggableContent content.
+ *
+ * Available variables:
+ * - content: A string containing a flagged contents content.
+ * - image_collection_count: an int containing the count of images on content.
+ * - video_collection_count: an int containing the count of videos on content.
+ */
+#}
+
+{# Output the FlaggableContent content. #}
+<p>{{ content }}</p>
+
+{# Output count of ImageCollection array. #}
+{% if image_collection_count > 0 %}
+  <p>
+    {% trans %}
+      {{ image_collection_count }} image.
+    {% plural image_collection_count %}
+      {{ image_collection_count }} images.
+    {% endtrans %}
+  </p>
+{% endif %}
+
+{# Output count of VideoCollection array. #}
+{% if video_collection_count > 0 %}
+  <p>
+    {% trans %}
+      {{ video_collection_count }} video.
+    {% plural video_collection_count %}
+      {{ video_collection_count }} videos.
+    {% endtrans %}
+  </p>
```

Comment:
I can give two quick examples and then we can discuss it:

`Bacon ipsum dolor sit amet spare ribs: Kielbasa sausage tongue boudin pastrami pork chop. Beef capicola shank, shankle jerky pig landjaeger doner ground round. Pig short loin meatloaf ham hock, turkey drumstick ground round pork chop turducken swine kevin venison pork loin tail. Rump ball tip flank pig beef chuck doner. Bresaola strip steak ball tip shankle shoulder sausage t-bone meatloaf meatball. (1 image) (2 videos).`

```
Bacon ipsum dolor sit amet spare ribs: Kielbasa sausage tongue boudin pastrami pork chop. Beef capicola shank, shankle jerky pig landjaeger doner ground round. Pig short loin meatloaf ham hock, turkey drumstick ground round pork chop turducken swine kevin venison pork loin tail. Rump ball tip flank pig beef chuck doner. Bresaola strip steak ball tip shankle shoulder sausage t-bone meatloaf meatball.
1 image.
2 videos.
```